etc/meGRE_bids_corrector.py

```diff
@@ -47,12 +47,12 @@
     #in the case of echo-1, the EchoNumber tag does not exist in the json file    
     if ("EchoNumber" not in json_data) and ("_echo_" in js):
         js_addEchoNum = re.sub("_echo_", "_echo-1_", js)
-        js_changeEnding = re.sub("GRE.*", "GRE.json", js_addEchoNum)
+        js_changeEnding = re.sub("_GRE[0-9]*", "GRE.json", js_addEchoNum)
```

Comment:
Would this be more safe? 
        js_changeEnding = re.sub("_GRE[0-9].json", "GRE.json", js_addEchoNum)

Reply:
sounds good, just added them. i guess we typically wouldn't expect any non json or nii files